make-debs.sh

```diff
@@ -34,19 +34,7 @@ git clean -dxf
 # d) contains the short hash of the commit
 #
 vers=$(git describe --match "v*" | sed s/^v//)
-#
-# creating the distribution tarbal requires some configure
-# options (otherwise parts of the source tree will be left out).
-#
-./autogen.sh
-./configure --with-rocksdb --with-ocf \
-    --with-nss --with-debug --enable-cephfs-java \
-    --with-lttng --with-babeltrace
-#
-# use distdir= to set the name of the top level directory of the
-# tarbal to match the desired version
-#
-make distdir=ceph-$vers dist
+./make-dist $vers
```

Comment:
the output file generated by `make-dist` is named `ceph-$ver.tar.bz2`. we might need to update following command lines accordingly:
```sh
mv ceph-$vers.tar.bz2 $releasedir/ceph_$vers.orig.tar.bz2
tar -C $releasedir -jxf $releasedir/ceph_$vers.orig.tar.bz2
```